scw/config.go

```diff
@@ -16,6 +18,72 @@ const (
 	defaultConfigPermission = 0600
 )
 
+const configFileTemplate = `# Scaleway configuration file
+# https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#scaleway-config
+
+# You need an access key and a secret key to connect to Scaleway API.
+# Generate your token at the following address: https://console.scaleway.com/account/credentials
+
+# An access key is a secret key identifier.
+{{ if .AccessKey }}access_key: {{.AccessKey}}{{ else }}# access_key: SCW11111111111111111{{ end }}
+
+# The secret key is the value that can be used to authenticate against the API (the value used in X-Auth-Token HTTP-header).
+# The secret key MUST stay secret and not given to anyone or published online.
+{{ if .SecretKey }}secret_key: {{ .SecretKey }}{{ else }}# secret_key: 11111111-1111-1111-1111-111111111111{{ end }}
+
+# Your organization ID is the identifier of your account inside Scaleway infrastructure.
+{{ if .DefaultOrganizationID }}default_organization_id: {{ .DefaultOrganizationID }}{{ else }}# default_organization_id: 11111111-1111-1111-1111-111111111111{{ end }}
+
+# A region is represented as a geographical area such as France (Paris) or the Netherlands (Amsterdam).
+# It can contain multiple Availability Zones.
+# Example of region: fr-par, nl-ams
+{{ if .DefaultRegion }}default_region: {{ .DefaultRegion }}{{ else }}# default_region: fr-par{{ end }}
+
+# A region can be split into many availability zones (AZ).
+# Latency between multiple AZ of the same region are low as they have a common network layer.
+# Example of zones: fr-par-1, nl-ams-1
+{{ if .DefaultZone }}default_zone: {{.DefaultZone}}{{ else }}# default_zone: fr-par-1{{ end }}
+
+# APIURL overrides the API URL of the Scaleway API to the given URL.
```

Comment:
IMO we should not expose this advanced feature by default (like `Insecure`)

Reply:
Exposing those features is not dangerous with sane defaults